back/app/models/nav_bar_item.rb

```diff
@@ -63,12 +76,14 @@ def set_code
   end
 
   def fallback_title_multiloc
-    key_code = custom? ? static_page.code : code
+    key_code = page? ? static_page.code : project? ? nil : code
```

Comment:
Ah so for projects there is no translation to look up write? In that case perhaps we should avoid trying `I18n.exists? key` as we already know it will return false. But not blocking.